app/components/ui/molecules/PeoplePicker.test.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import { ThemeProvider } from 'styled-components'
+import theme from '@elifesciences/elife-theme'
+import PeoplePicker from './PeoplePicker'
+
+const people = [
+  { id: 1, name: 'Annie' },
+  { id: 2, name: 'Bobby' },
+  { id: 3, name: 'Chastity' },
+  { id: 4, name: 'Dave' },
+]
+
+const makeWrapper = props =>
+  mount(
+    <ThemeProvider theme={theme}>
+      <PeoplePicker
+        // eslint-disable-next-line react/no-children-prop
+        children={innerProps => <PeoplePicker.Body {...innerProps} />}
+        onCancel={jest.fn()}
+        onSubmit={jest.fn()}
+        people={people}
+        {...props}
+      />
+    </ThemeProvider>,
+  )
+
+const getPersonPodButton = (wrapper, index) =>
+  wrapper.find('button[data-test-id="person-pod-toggle"]').at(index)
+
+const expectSelectionLength = (wrapper, length) =>
+  expect(wrapper.find('SelectedItem')).toHaveLength(length)
+
+describe('PeoplePicker', () => {
+  it('calls child render prop', () => {
+    const render = jest.fn(() => null)
+    makeWrapper({ children: render })
+    expect(render).toHaveBeenCalled()
+  })
+
+  it('renders one person pod per person', () => {
```

Comment:
This is probably extremely pedantic, but the wording made me think that you were actually going to check whether there was a single pod for "Annie" pod & a single pod for "Bobby", whereas this test would still pass even if 4 "Annie"s were rendered.

How about "renders as many pods as there are people"? Again, this may feel overly nitpick-y, but if it made me think it was doing something other than it was doing, it might just help someone who is using the failing test's message down the line? :woman_shrugging: